java/geodesic-operations/README.md

```diff
@@ -0,0 +1,33 @@
+# Geodesic Operations 
+Demonstrates how to use the GeometryEngine to calculate a geodesic path between two points and measure its distance.
+
+![Geodesic Operations Image](GeodesicOperations.png)
+
+## How to use the sample
+Click on the map to select a path destination. The geodesic path between the two points will update.
+
+## How it works
+1. Create a `Polyline` using two points.
+1. Pass this polyline to: `GeometryEngine.densifyGeodetic(polyline, segmentLength, unitOfMeasurement, GeodeticCurveType
+    .GEODESIC)`. This will create a new polyline with segments of length `segmentLength` and 
+    `LinearUnit` set to `unitOfMeasurement`. The curve will be geodesic.
+1. You can set this geometry to a `Graphic` to display the curve in a `GraphicsOverlay`.
+1. To get the distance, use `GeometryEngine.lengthGeodetic(pathGeometry, unitOfMeasurement, GeodeticCurveType.GEODESIC)`  
+
+
```

Comment:
The common samples team is in the process of standardizing our template as per: https://devtopia.esri.com/runtime/common-samples/wiki/Standard-sample-documentation-template-%28ReadMe.md%29#sample-title

Please rename Features -> Relevant API